metaci/build/models.py

```diff
@@ -321,7 +326,21 @@ def run(self):
 
             # Initialize the project config
             project_config = self.get_project_config()
-
+            if self.plan.role == "push_sandbox" or self.plan.role == "push_production":
```

Comment:
Can we put the logic for determining whether to call CTC into its own method (maybe `should_send_start_stop`) -- then we can use that both here and on line 445 to make sure it's consistent (you only have push_sandbox down there)

Reply:
Also now I'm thinking a better way to determine whether to make these calls is if the org is named `packaging`. That way we'll be sure to require CTC checks for any plan that runs against production.

We should add a Django setting based on an environment variable for whether to enable this feature. That way we can merge it even if we're not ready to enable it in production yet.